theseus/geometry/functorch.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import threading
+from typing import Any
+
+
+class _FunctorchContext:
```

Comment:
Given the name intended name change for the context we should likely change the name for this file too.